src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -187,5 +187,33 @@ public static class TimeSeriesCatalog
             new SsaForecastingEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnName, inputColumnName, windowSize, seriesLength, trainSize,
                 horizon, isAdaptive, discountFactor, rankSelectionMethod, rank, maxRank, shouldStabilize, shouldMaintainInfo, maxGrowth, confidenceLowerBoundColumn,
                 confidenceUpperBoundColumn, confidenceLevel, variableHorizon);
+
+        /// <summary>
+        /// Create <see cref="SrCnnEntireAnomalyEstimator"/>, which detects timeseries anomalies for entire input using SRCNN algorithm.
+        /// </summary>
```

Comment:
Can we add some references to the algorithm? Any published papers or wikipedia articles?

Reply:
> Can we add some references to the algorithm? Any published papers or wikipedia articles?

Added.

Reply:
Sorry, I don't see the references. Can you please point them out?

---
In reply to: [418089729](https://github.com/dotnet/machinelearning/pull/4998#discussion_r418089729) [](ancestors = 418089729)

Reply:
Sorry to confuse you, the algorithm description is added in SrCnnEntireAnomalyEstimator.cs ln28 to ln 56.

---
In reply to: [418259971](https://github.com/dotnet/machinelearning/pull/4998#discussion_r418259971) [](ancestors = 418259971,418089729)